reportportal_client/service.py

```diff
@@ -126,6 +128,9 @@ def __init__(self, endpoint, project, token, api_base="api/v1",
                                  self.project)
 
         self.session = requests.Session()
+        if retries:
+            self.session.mount('https://', HTTPAdapter(max_retries=retries))
```

Comment:
Don't know why you need to bind the adapter twice.
Could you please explain, @pshv ?

Reply:
requests.Session() really have these two .mount() calls but with default (empty) parameter sets for HTTPAdapter:

`sessions.py -> class Session -> def __init__`
```
self.adapters = OrderedDict()
self.mount('https://', HTTPAdapter())
self.mount('http://', HTTPAdapter())
```
And because of that we can't pass any additional params to HTTPAdapter without changing requests lib. Also, if we'd take a look at mount() method implementation, the main thing it does is actually overrides old adapter (from session init method, without max_retries param) with our's updated version.